lib/mixins.js

```diff
@@ -45,6 +45,7 @@ MixinProvider.prototype.applyMixin = function applyMixin(modelClass, name, optio
       modelClass.mixin(model, options);
     } else {
       debug('Invalid mixin: %s', name);
+      throw new Error('Invalid mixin: ' + name);
```

Comment:
Please include the model name in the error message, otherwise it's impossible to find the model which is referencing this missing mixing.

I would also reword the message a bit, e.g.

```js
'Model ' + modelClass.modelName + ' uses unknown mixin ' + name
```